packages/ppx_protocol_conv/ppx_protocol_conv.2.0.0/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "1.2"
+maintainer: "Anders Fugmann <anders@fugmann.net>"
+authors: "Anders Fugmann"
+homepage: "https://github.com/andersfugmann/ppx-deriving-protocol"
```

Comment:
@andersfugmann It looks like these links all point to a deprecated repository.

Reply:
Thanks. Nice catch.